app/src/main/shared/components/UploadFileForm.tsx

```diff
@@ -76,7 +75,7 @@ export class UploadFileForm extends React.Component<UploadFileProps, UploadFileS
                     <label className="customFileUpload">
                         <input name="file" type="file" onChange={this.handleChange.bind(this)}/>
                         <div className="button mt-2 mb-2">
-                            {this.props.uploadText}
```

Comment:
Decided there was no point in this being configurable, it should always just say "Choose file"